### PR TITLE
[CAS-288] Fixing coroutines test

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
### Description

Fixing unit tests for Coroutines in `StartStopBufferTest`. 

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
- [x] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[*.{kt,kts}]
+disabled_rules=import-ordering
```
Comment (zsmb13) [inline .editorconfig:2]: What tool does this affect, ktlint? Why do we _not_ want our imports sorted?

**Review by zsmb13 — CHANGES_REQUESTED** [review]
The coroutine changes look all good, just have a question about the editor config change

File: .editorconfig (hunk 1 above)
Comment (leandroBorgesFerreira) [reply .editorconfig]: Yes, Ktlint. Removed.

**Review by zsmb13 — APPROVED** [review]
I give up trying to understand what's happening with the import order, but the coroutine changes are still good to go 😄 👍

**Pull request merged**